cmd/bootkube/render.go

```diff
@@ -100,7 +100,8 @@ func validateRenderOpts(cmd *cobra.Command, args []string) error {
 		return errors.New("You must specify either all or none of --etcd-ca-path, --etcd-certificate-path, and --etcd-private-key-path")
 	}
 	if renderOpts.etcdCertificatePath != "" && renderOpts.selfHostedEtcd {
-		return errors.New("Cannot specify --etcd-certificate-path with --experimental-self-hosted-etcd")
+		return errors.New("Cannot specify --etcd-certificate-path with --experimental-self-hosted-etcd." +
+			" Self-hosted etcd + TLS will auto-generate certs based on root CA cert.")
```

Comment:
Can we move forward this PR?
I don't think this comment is relevant to this PR.